ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh

```diff
@@ -53,9 +53,13 @@ fi
 
 cd "\${REPO_DIR}"
 
+cat << VARS >> /root/config
+export DISCONNECTED="${DISCONNECTED:-}"
```

Comment:
I think it will not work for you, as ``DISCONNECTED`` is not a parameter of the gather step
why not reorder the lines in the operator-setup step?

Reply:
not a parameter meaning it's [here](https://github.com/openshift/release/blob/8230ca3569f1030a448dcc74f9c07f1f3e630931/ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-ref.yaml#L10) but not [there](https://github.com/openshift/release/blob/29112064321b0bc26611ef7aba5a925fbe45ee18/ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-ref.yaml)

Reply:
Probably I dont understand how the flow goes, but I dont see how operator-setup step is relevant for this

Wouldnt it be cleaner to just add DISCONNECTED as an env to the gather step, as it is for the assisted-baremetal-operator-ztp step?

Reply:
maybe
but for that you'll need to add it here: https://github.com/openshift/release/blob/29112064321b0bc26611ef7aba5a925fbe45ee18/ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-ref.yaml